### DISCUSSION TITLE
Battery

### DISCUSSION THREAD

**Discussion by BenGWeeks** [cryptoadvance/specter-diy#188] [Q&A]
Can you please recommend a battery (that will also fit in the Specter-DIY case)?

**Comment by stepansnigirev** [comment]
Do you mean for barebones DIY, or for the one with a shield?
For barebones I don't think there are power banks that are small enough to fit in the enclosure, at least I wasn't able to find one that is reasonably small.
If you want to use an actual battery with VIN PIN you need to provide at least 6V to it - so you will need a power booster, 9V battery like coldcard cold power, or connect several AA/AAA batteries in series.
Keep in mind that 9V battery will be discharged very quickly because of the terrible design of the power circuit in the discovery board.
